servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/NoopAsyncContextProvider.java

```diff
@@ -136,42 +152,42 @@ public ScheduledExecutorService wrapJdkScheduledExecutorService(final ScheduledE
 
     @Override
     public <T> CompletableFuture<T> wrapCompletableFuture(final CompletableFuture<T> future,
-                                                          final ContextMap context) {
+                                                          final CapturedContext context) {
         return future;
     }
 
     @Override
-    public Runnable wrapRunnable(final Runnable runnable, final ContextMap context) {
+    public Runnable wrapRunnable(final Runnable runnable, final CapturedContext context) {
         return runnable;
     }
 
     @Override
-    public <V> Callable<V> wrapCallable(final Callable<V> callable, final ContextMap context) {
+    public <V> Callable<V> wrapCallable(final Callable<V> callable, final CapturedContext context) {
         return callable;
     }
 
     @Override
-    public <T> Consumer<T> wrapConsumer(final Consumer<T> consumer, final ContextMap context) {
+    public <T> Consumer<T> wrapConsumer(final Consumer<T> consumer, final CapturedContext context) {
         return consumer;
     }
 
     @Override
-    public <T, U> Function<T, U> wrapFunction(final Function<T, U> func, final ContextMap context) {
+    public <T, U> Function<T, U> wrapFunction(final Function<T, U> func, final CapturedContext context) {
         return func;
     }
 
     @Override
-    public <T, U> BiConsumer<T, U> wrapBiConsumer(final BiConsumer<T, U> consumer, final ContextMap context) {
+    public <T, U> BiConsumer<T, U> wrapBiConsumer(final BiConsumer<T, U> consumer, final CapturedContext context) {
         return consumer;
     }
 
     @Override
     public <T, U, V> BiFunction<T, U, V> wrapBiFunction(final BiFunction<T, U, V> func,
-                                                        final ContextMap context) {
+                                                        final CapturedContext context) {
         return func;
     }
 
-    private static final class NoopContextMap implements ContextMap, Scope {
+    static final class NoopContextMap implements ContextMap {
```

Comment:
Why this one doesn't need `Scope` and `CapturedContext`? Doesn't cost us much to add and no need to worry about any allocations with AsyncContext is disabled